cheqd/cheqd/anoncreds/registry.py

```diff
@@ -179,7 +183,6 @@ async def register_schema(
             LOGGER.debug("Existing schema %s", existing_schema)
             # update if schema exists
             if existing_schema is not None:
-                LOGGER.debug("UPDATING SCHEMA")
                 cheqd_schema = ResourceUpdateRequestOptions(
                     options=Options(
                         name=resource_name,
```

Comment:
@DaevMithran  Does "updateResource" work when the same name and type is passed?

Reply:
The revocation tests already handle it. Added duplicate test for schemas too